src/fsmc_lcd/pins.rs

```diff
@@ -0,0 +1,763 @@
+        (
+            Lcd {
+                _sub_bank: PhantomData,
+            },
+            Lcd {
+                _sub_bank: PhantomData,
+            },
+        )
+    }
+}
+impl sealed::Conjure for (Lcd<SubBank3>, Lcd<SubBank4>) {
+    fn conjure() -> Self {
+        (
+            Lcd {
+                _sub_bank: PhantomData,
+            },
+            Lcd {
+                _sub_bank: PhantomData,
+            },
+        )
+    }
+}
+impl sealed::Conjure for (Lcd<SubBank1>, Lcd<SubBank2>, Lcd<SubBank3>) {
+    fn conjure() -> Self {
+        (
+            Lcd {
+                _sub_bank: PhantomData,
+            },
+            Lcd {
+                _sub_bank: PhantomData,
+            },
+            Lcd {
+                _sub_bank: PhantomData,
+            },
+        )
+    }
+}
+impl sealed::Conjure for (Lcd<SubBank1>, Lcd<SubBank2>, Lcd<SubBank4>) {
+    fn conjure() -> Self {
+        (
+            Lcd {
+                _sub_bank: PhantomData,
+            },
+            Lcd {
+                _sub_bank: PhantomData,
+            },
+            Lcd {
+                _sub_bank: PhantomData,
+            },
+        )
+    }
+}
+impl sealed::Conjure for (Lcd<SubBank1>, Lcd<SubBank3>, Lcd<SubBank4>) {
+    fn conjure() -> Self {
+        (
+            Lcd {
+                _sub_bank: PhantomData,
+            },
+            Lcd {
+                _sub_bank: PhantomData,
+            },
+            Lcd {
+                _sub_bank: PhantomData,
+            },
+        )
+    }
+}
+impl sealed::Conjure for (Lcd<SubBank2>, Lcd<SubBank3>, Lcd<SubBank4>) {
+    fn conjure() -> Self {
+        (
+            Lcd {
+                _sub_bank: PhantomData,
+            },
+            Lcd {
+                _sub_bank: PhantomData,
+            },
+            Lcd {
+                _sub_bank: PhantomData,
+            },
+        )
+    }
+}
+impl sealed::Conjure for (Lcd<SubBank1>, Lcd<SubBank2>, Lcd<SubBank3>, Lcd<SubBank4>) {
+    fn conjure() -> Self {
+        (
+            Lcd {
+                _sub_bank: PhantomData,
+            },
+            Lcd {
+                _sub_bank: PhantomData,
+            },
+            Lcd {
+                _sub_bank: PhantomData,
+            },
+            Lcd {
+                _sub_bank: PhantomData,
+            },
+        )
+    }
+}
+
+/// One, two, three, or four chip select pins
+///
+/// Due to trait system limitations, this trait is only implemented for pins wrapped in the
+/// `ChipSelect1`, `ChipSelect2`, `ChipSelect3`, and `ChipSelect4` wrappers.
+///
+/// This trait is implemented for all non-empty subsets of the 4 possible chip select signals.
+/// The pins must be in order.
+///
+/// # Example types that implement `ChipSelectPins`
+///
+/// Wrapped single pins:
+/// * `ChipSelect1<PD7<Alternate<AF12>>>`
+/// * `ChipSelect2<PG9<Alternate<AF12>>>`
+/// * `ChipSelect3<PG10<Alternate<AF12>>>`
+/// * `ChipSelect4<PC4<Alternate<AF12>>>`
```

Comment:
According to the implementation `PC4` is only usable for some models while the more general one is `PG12`.